submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+When I came across a glowing green patch of Uranium ore for the first time, I saw that Factorio had some juicy stuff waiting for me somewhere down the tech tree: Nuclear energy! While in today’s world the trials and triumphs of nuclear power are hotly debated, the Factorio community similarly has its ongoing discussions on whether it is worthwhile: Going nuclear is a couple steps up in complexity from the other power options, while also it is completely optional on the path to launching a rocket. Hence many players tend to not bother with diving into it, or just find a good blueprint as a blackbox and call it a day. Others hear of the warnings of the nuclear reactor as a UPS killer, or even a literal killer when it explode dramatically, and they decide to steer clear of it. Yes, there is a lot to look out for, but you can trust in Wube: Like many other carefully balanced mechanics of the game, nuclear power is reasonably easy to get into and you can quickly make something that just works. As for the more complicated and colorful part, I have a few guidelines to offer from my own fair share of tinkering and the findings of the glowing gems of our community.
+
+
+
+## Demystefying Nuclear Power
+
+While it may seem offputing with a whole bunch of new machines to it, at its core, a vanilla nuclear plant setup is only a little different from a burner steam power setup: You have a fuel being burned, heat creating steam, and an engine making pwoer from it. The extra complexity comes from three elements: Firstly, the fuel used in reactors is unique and is made in multiple steps instead of using the miner output directly. The official wiki has a [handy tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) that covers all the basics such as mining and centrifuging uranium ore, as well as setting up a basic single-reactor plant. 
+
+Secondly, nuclear reactors do not stop burning fuel no matter what, and this can lead to a rapid heat buildup, although this *usually* is not as bad as it sounds, as I will clarify shortly. 
+
+Thirdly, heat and fluid management become a couple steps more complex (but ultimately still manageable) if you decide to combine multiple reactors, but that is also where immense efficiency gains can be made. The magic comes from the neighbor bonus: if any two running reactors share a full edge with each other, both reactors get an additional 40MW of heat output. With this being possible on up to 4 edges, you to get up to 500% of the original power output for the same amount of fuel! These enticing efficiency numbers are what push players to get inventive about managing the massive amounts of heat that is generated in huge nuclear reactor arrays. Hence we have a system that is reasonably easy to set up, and pretty interesting to scale up and optimize.

```

Comment:
"Thirdly, heat and fluid management become a couple steps more complex, but usually very managable unless if you decide to combine multiple reactors,"

Also, I don't think the explanation of neighbouring bonusses belong in the bullet point. You could maybe mention neighbouring bonusses in the first paragraph of this section, and then here say that fluids become hard to manage in big nuclear reactors because they become so powerful.